trade-finance-manager-ui/server/controllers/helpers/isFacilityEndDateEnabledForFacility.js

```diff
@@ -0,0 +1,13 @@
+import { isTfmFacilityEndDateFeatureFlagEnabled, MAPPED_FACILITY_TYPE } from '@ukef/dtfs2-common';
+
+/**
+ * Facilities are part of a GEF deal if and only if they have a cash or contingent facility type.
+ * In mapGefFacility.js, the received facility gets mapped from 'CASH' or 'CONTINGENT' to 'Cash facility' and 'Contingent facility'
+ */
+const isFacilityPartOfGefDeal = (type) => {
+  return type === MAPPED_FACILITY_TYPE.CASH || type === MAPPED_FACILITY_TYPE.CONTINGENT;
+};
+
+export const isFacilityEndDateEnabledForFacility = (facility) => {
```

Comment:
this could be named to `isFacilityEndDateEnabled`.  The ForFacility seems unnecessary as facility already mentioned

Reply:
The concept of the parameter `facilityEndDate` lives in other items other than `facility` so I think we should clarify it here to self document